test/e2e/index.test.ts

```diff
@@ -257,5 +257,63 @@ describe('e2e', () => {
         ),
       ).to.equal(scrollTop);
     });
+
+    it('should edit date cells', async () => {
+      await renderFixture('DataGrid/KeyboardEditDate');
+
+      // Edit date column
+      expect(
+        await page.evaluate(
+          () => document.querySelector('[role="cell"][data-field="birthday"]')!.textContent!,
+        ),
+      ).to.equal('2/29/1984');
+
+      // set 06/25/1986
+      await page.dblclick('[role="cell"][data-field="birthday"]');
+      await page.keyboard.press('0');
+      await page.keyboard.press('6');
+      await page.keyboard.press('2');
+      await page.keyboard.press('5');
+      await page.keyboard.press('1');
+      await page.keyboard.press('9');
+      await page.keyboard.press('8');
+      await page.keyboard.press('6');
```

Comment:
Maybe it would be easier to read with https://playwright.dev/docs/api/class-page#page-type